client/jetpack-connect/test/auth-form-header.js

```diff
@@ -33,9 +33,9 @@ const DEFAULT_PROPS = {
 
 describe( 'AuthFormHeader', () => {
 	test( 'renders as expected', () => {
-		const wrapper = shallow( <AuthFormHeader { ...DEFAULT_PROPS } /> );
+		const { container } = render( <AuthFormHeader { ...DEFAULT_PROPS } /> );
 
-		expect( wrapper ).toMatchSnapshot();
+		expect( container ).toMatchSnapshot();
```

Comment:
Should it be `container.firstChild` to not include the RTL wrapper in the snapshot? Not a blocker.

Reply:
I guess it doesn't matter that much, it's essentially just one more or less div inside the snapshot.